packages/core/src/serialization.ts

```diff
@@ -35,11 +35,13 @@ export type SerializedReference = {
  * Type definition for an AST node serialized to JSON.
  */
 export type SerializedNode = {
+    // TODO should be Id
     id: string
     classifier: MetaPointer
     properties: SerializedProperty[]
     containments: SerializedContainment[]
     references: SerializedReference[]
+    // TODO inconsistent with the use of Ids
```

Comment:
You mean: for consistency, we should fix the 1st TODO, and then both are?

Reply:
Inconsistency is between using `Id[]` and `Ids` and `string`. I personally would stick with `Id` and `Id[]`, we use `[]` for other array types (like `SerializedProperty `, etc. ) as well. `Ids` is just obfuscating things.  
So we should change the `string` in the first TODO into `Id` and replace `Ids` (line 21) with `Id[]`.

Reply:
fixed